apps/web/src/app/api/cron/short-url-cleanup/route.ts

```diff
@@ -0,0 +1,28 @@
+import { prisma } from '@repo/db';
+import type { NextRequest } from 'next/server';
+
+export async function GET(request: NextRequest) {
+  const authHeader = request.headers.get('authorization');
+  if (authHeader !== `Bearer ${process.env.CRON_SECRET}`) {
```

Comment:
one thing i did for DX on the role sync was disable the secret check in dev mode:

https://github.com/typehero/typehero/blob/main/apps/web/src/app/api/cron/sync-contributors/route.ts#L12

Reply:
Oh Nice.. Will Add as well